helm/aws-load-balancer-controller/values.yaml

```diff
@@ -284,6 +284,11 @@ backendSecurityGroup:
 # disableRestrictedSecurityGroupRules specifies whether to disable creating port-range restricted security group rules for traffic
 disableRestrictedSecurityGroupRules:
 
+# featureGates set of key: value pairs that describe AWS load balance controller features
+featureGates: {}
```

Comment:
Lets group under controllerConfig, we plan to group together related values in future, we will start with the new configurations. Something like this:
 
```
# controllerConfig specifies controller configuration
controllerConfig:
    # featureGates set of key: value ...
    featureGates: {}
        # ...
```

Reply:
Done

Reply:
@kishorj